features/attached_commands.feature

```diff
@@ -24,6 +35,37 @@ Feature: Command behaviour when attached to an UA subscription
            | hirsute |
            | groovy  |
 
+    @series.xenial
+    @series.bionic
+    @series.focal
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attached refresh config in a ubuntu machine vm
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `snap install canonical-livepatch` with sudo
+        And I run `snap set system proxy.https=localhost:1234` with sudo
+        And I run `canonical-livepatch config http-proxy=localhost:12345` with sudo
+        And I run `ua refresh config` with sudo
+        Then I will see the following on stdout:
+            """
+            We noticed you have some system proxies set but did not configure them
+            with ubuntu-advantage-tools. If you wish to remove them, run the following
+            commands:
+
+                snap unset system proxy.https
+                canonical-livepatch config http-proxy=
+
+            Or, if you want ua to stop showing this message, you can add the proxy
+            information to /etc/ubuntu-advantage/uaclient.conf
+
+            Successfully processed your ua configuration.
```

Comment:
@mitechie @blackboxsw @lucasmoura 

This is my draft of what we could display in the scenario when
1. there is a snap/livepatch proxy set
2. uaclient.conf does not have any proxy information
3. the user runs `ua refresh config`

I'm open to any changes.

Reply:
I'm working to find a way to update the spec for this but I think I'm -1 on this approach. If there's nothing in the config file then refresh config would no-op and we'd not offer to clear proxies or the like. 

We did agree that when use ua config set <x> we'd note any conflicts we'd be overriding, but I think that's all. In that light, during unset we'd simply remove the proxy values in place. 

With the refresh of the config the user hasn't said if they mean "no value" or an explicit "empty value" and so I think that's just a no-op.

Reply:
Refreshing config:

No proxy settings set in config, however, there is proxy config on the livepatch service. See HTTP://... for more information on proxy config.

Reply:
or should that be less about livepatch and more about snaps and pointing to snap proxy documentation vs our own